CONTRIBUTORS.yaml

```
@@ -655,6 +655,7 @@ Sch-Da:
     bio: Researcher at Galaxy Europe
     affiliations:
       - uni-freiburg
+      - dekcd
```

(capitalization should match ORGANISATIONS.yaml file)

```suggestion
      - deKCD
```